subsys/nrf_rpc/include/nrf_rpc_utils.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef UTILS_NRF_RPC_UTILS_H_
+#define UTILS_NRF_RPC_UTILS_H_
+
+/**
+ * @defgroup nrf_rpc_utils nRF RPC utilities.
```

Comment:
I restate that IMO the name "nRF RPC utils" is too vague. It should be clear for future developers the purpose of this group is and which nRF RPC commands fit in this group. "nRF RPC utils" could equally mean utilities for using nRF RPC library.